installations/InstallationsSnippets/AppDelegate.swift

```diff
@@ -38,6 +38,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
     // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
   }
+    
+  func handleInstallationIDChange() {
+    // [START handle_installation_id_change]
+    NotificationCenter.default.addObserver(forName: .InstallationIDDidChange,
+                                            object: nil,
+                                             queue: nil) { (notification) in
```

Comment:
I think we should recommend weak capturing of self to avoid retaining `self` by the notification center. The final recommended snippet with the recommendations:
```
var installationIDObserver: NSObjectProtocol?
func handleInstallationIDChange() {
    // [START handle_installation_id_change]
    installationIDObserver = NotificationCenter.default.addObserver(forName: .InstallationIDDidChange,
                                            object: nil,
                                             queue: nil) { [weak self] (notification) in
      // Fetch new Installation ID
      self?.fetchInstallationToken()
    }
    // [END handle_installation_id_change]
  }
```

Reply:
done